### PR TITLE
VIH-10350  Decommission obsolete feature flags

### PR DESCRIPTION
**Pull request by marcogagliardi**
### Jira link (if applicable)
[VIH-10350](https://tools.hmcts.net/jira/browse/VIH-10350)

### Change description ###
deleted code for FeatureToggleService

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: UserApi/UserApi.Common/Configuration/FeatureToggles.cs
Comment (shaed-parkar) [inline UserApi/UserApi.Common/Configuration/FeatureToggles.cs:1]: Don't forget to remove launch darkly from the nuget package references

Comment (marcogagliardi) [reply UserApi/UserApi.Common/Configuration/FeatureToggles.cs:1]: done

**Review by shaed-parkar — APPROVED** [review]

**Review by ronaldmalone — APPROVED** [review]

**Pull request merged**